run-saturn.sh

```
@@ -0,0 +1,33 @@
+    fi
+    
+    export OPENROUTER_API_KEY="$api_key"
+    echo "✅ API key set for this session"
+    echo ""
+fi
+
+echo "🚀 Launching Saturn..."
+echo ""
+
+# Run Saturn
+cd /Users/jlee/projects/_tools_/Saturn
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)